### PR TITLE
Change Box Bar Maint into a more flavourful alternative

### PR DESCRIPTION
**Pull request by ghost**
- Two new areas added, Ghetto Theatre and Ghetto Morgue
- Auxiliary Charging icon has finally been fixed

I found the new Box Bar Maintenance to have something wrong and dissonant with the rest of the station, from the cult classic Ghetto Bar to the small empty rooms in Science Maint. After a bit of consulting, I estimated the main problem is that a request for more Maintenance was misinterpreted as a request for Deff Maint (aka dump random rooms)

As a reason, I have completely redrawn that area, with the following changes : 
- The monkey pen has been completely removed and the area it occupied flattened and redrawn
- A new Ghetto Theatre has been added. It's a loose mix of old Bar Theatre and New Courtroom. It never opened due to a noodle incident, although there's a few things you can grab from here. It has an APC that is not wired properly and has no battery, like the Ghetto Bar
- A new Ghetto Morgue has been added. Basically a Morgue with Chaplain/Maint access (same as North Chapel Maint Entrance). No outstanding loot in here, although it is secluded and has both morgue trays and coffins
- The rest of the Maint was reshaped. Due to the awkward tunnel nature of the old Bar Maint, it was hard to graft all the pipes without ending up with a one-tile wide tunnel left-most. The rest I feel was fitted about right. The only room that has disappeared from "Original Bar Maint" was that big empty office room
- Other changes outlined below

Screenshot : 
newboxbarmaint

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
Can't see shit but doesn't sound good.

**Comment by TheFriggenSawbones** [comment]
the slot machine will be missed

**Comment by Ephx1** [comment]
Is Box going to be the new deff at this rate?

**Comment by PJB3005** [comment]
Dylan hide the area layer ffs.

**Comment by Shadowmech88** [comment]
Personally I think any maintenance expansions should be big chunks just tacked onto the side of the existing station, like giant space debris that collided with it.
I don't like the idea of changing the structure of the station's own maintenance too much.

**Comment by Kitfox88** [comment]
> Personally I think any maintenance expansions should be big chunks just tacked onto the side of the existing station, like giant space debris that collided with it.

space hulk station and mode when

**Comment by CptWad** [comment]
Aw but I liked my changes.
Also why a ghetto morgue?
I don't understand the point of a place meant to put dead bodies with immovable objects that only idiots would store their kills in.
I mean its too obvious a spot to put dead people and really only serves as a free supply of caskets for vampires, which they could get easy as fuck anyway

**Comment by Imagebard** [comment]
:-1: please don't merge

**Comment by gbasood** [comment]
Please give us a screenshot WITHOUT areas.

**Comment by BestName** [comment]
Why are you giving box a ghetto theater when it doesn't even have a real theater?

**Comment by ghost** [comment]
Alright, let's see
- Box has a Theater, it's where the Clown and Mime spawn
- If specific loot/machinery like the one armed bandit or the toy captain felt nice to add, manifest it and I'll patch them in

The rest of the feedback I don't understand, especially the one comment accusing me of doing Deff maint when it's exactly what I am trying to swap out

newboxbarmaint

**Comment by NigglyWiggly** [comment]
WHERE'S THE SCREENSHOT DYLAN

**Comment by ghost** [comment]
It's in the OP @nigglywiggly

**Comment by NigglyWiggly** [comment]
THE ONE WITHOUT THE AREA LAYER IDIOT

**Comment by ghost** [comment]
Refresh the page

**Comment by ghost** [comment]
Are you thick? It's right there. Did you even look when I said the OP?

**Comment by BestName** [comment]
> Box has a Theater, it's where the Clown and Mime spawn

That's the backstage, there is no theater.

**Comment by 9600bauds** [comment]
:+1:
If the structure of Maint is to be changed it should at least pretend to blend in.
Only complaint is that maybe the abandoned bar should be closer to the station, maybe by switching around the radio transmitter and such.

**Comment by NigglyWiggly** [comment]
@dylanstrategie 
Ah, thanks.

**Comment by gbasood** [comment]
I do prefer this over monkey maint :+1:

**Comment by ghost** [comment]
I'm going to drop in a few things that were missing like the toys and the one armed bandit

I'll take a look at the Theatre to see if I can move it around a bit

**Comment by NigglyWiggly** [comment]
Looks good

Now just remove the Snowflake gashapon oh god it's embarrassing that thing was adminspawn for a reason and now it's so close to the library

**Comment by ghost** [comment]
New draft, following changes : 
- Added one-armed bandit and Captain toy in Theatre
- Named most airlocks and doublechecked Ghetto Morgue access
- Moved Theatre further down, reshaped most of the area again to be more aerated and more logical, aka feels like it could have been part of the station if finished
- Media Server (name of new airlock) has a high-security airlock (graphical only) with Telecomms access, to prevent greyshits from stealing the server

Screenshot : 
newboxbarmaint

**Comment by dylanstrategie** [comment]
Wrong screenshot, one second

**Comment by NigglyWiggly** [comment]
> Snowflake is still there

_reeeeeee_

**Comment by 9600bauds** [comment]
:+1::x::100:
I'm not sure if I understand the airlock for the Radio Transmitter though, it's a secure airlock but only in sprite?

**Comment by Imagebard** [comment]
Snowflake? You mean the Librarian Doll on the 3x3 maint square, above the Cape?

**Comment by NigglyWiggly** [comment]
Yes

**Comment by gbasood** [comment]
I'm actually not a huge fan of the area on the upper right, now that I think about it. At least as a concept. Interesting for an antag chaplain though.

**Comment by ghost** [comment]
It's supposed to be kind of a flavor thing. The Chaplain can access it via the Chapel, although the access is intentionally convoluted and dangerous because this part of the station was never finished

It's kind of the Chaplain's private Morgue and (discount) Shrine place otherwise

**Comment by CptWad** [comment]
The two tile wide hallway with nothing in it to the left is a bit empty
Also if you didn't put the monkey back in just slap it in the chefs meat locker
Because I am a tremendous faggot heres some stuff I will recommend
my shit

Otherwise the extra morgue bit is weird and a tad out of place, given that everything else is barebones or run down

**Comment by Shadowmech88** [comment]
Why would the chaplain _need_ a private morgue though? I can't picture anyone ever using it.

**Comment by ghost** [comment]
I'm kinda split on the propositions here
- The door in the middle hallway would actively prevent the Chaplain from reaching his backup Morgue, and it's supposed to be highly unfinished. In MUH LORE, the NT Engineers noticed the layout is fucked up at that point, especially with all the piping
- Tinted glass panel is a bit useless, and normal window into the room defeats the point of the "secluded morgue room"
- The bottom left part is intended to be kinda open ended, was supposed to be an open area, again in muh lore
- Making the top left corner smaller feels awkward. I guess an extra thing or two might be nice

@Shadowmech88 I'm sure the Chaplain players will figure an use for a more permanent Morgue that they can access. In that way it's kind of a more private recluse

**Comment by CptWad** [comment]
- The chaplain has maint access
- While I will admit my secluded monkey room was worse, having a room completely dedicated to a single job is redundant and alienates all the assistants who will want to use it
- Tinted glass looks good, also who wants to stare at dead bodys? You can remove them if you happen to anyways
- The left side is way too empty and if you dont like my suggestion at least add a few crates you scrapped since those are supposed to be there for cargo to find

**Comment by ghost** [comment]
I'd honestly prefer to avoid closing up the middle, and now that I look at it again the tinted glass would block off the morgue trays too. So less than ideal

I'll add a few more things to the left hallway

**Comment by CptWad** [comment]
wait hold on
why are there THREE sets of liquid tanks
did I do that first?

**Comment by CptWad** [comment]
Yeah no I didn't
Why are there now three sets of the liquids? For what purpose?

**Comment by dylanstrategie** [comment]
I'll remove one then

**Comment by ghost** [comment]
Thinking about it, the Botanists might like all the extra water for VERY long rounds

**Comment by CptWad** [comment]
there's infinite water in the sinks

**Comment by ghost** [comment]
I'll prune the <s>third</s> fourth water tank then

**Comment by 9600bauds** [comment]
Maint Morgue would be a pretty sick vamp crib

Otherwise, we still lack Maint Botany in Box

**Comment by ghost** [comment]
Hopefully last draft
- Removed a bunch of tanks
- Added some clutter to the top-left hallway
- Added a few girders to close up the Maintenance a bit more, middle hallway still has a way through

newboxbarmaint

**Comment by 9600bauds** [comment]
>wet floor signs and vomit
hhehehe
I like it

**Comment by 9600bauds** [comment]
Beachball in Maint is completely OP though

**Comment by ghost** [comment]
Time to get those Beach valids

**Comment by CptWad** [comment]
fuck off wad
well rip my monkey room
anyway ill give this a :+1: if you replace the closets on the left with crates

**Comment by ghost** [comment]
Alright, time to last last draft then

**Comment by NigglyWiggly** [comment]
Remove snowflake pls

**Comment by ghost** [comment]
You want the Librarian statuette removed ? What for ?

**Comment by NigglyWiggly** [comment]
It's evidence of the horrible things I've added to the code.
I'll also probably be commenting it out the next time I update gashapon stuff

**Comment by 9600bauds** [comment]
yes remove

**Comment by ghost** [comment]
newboxbarmaint
- Removed statuette
- Changed closets to crates
- Added a little meaningless loot. A health analyzer, a penlight, and a liver

**Comment by Duny-** [comment]
TOO MUCH MAINT SHIT MAP

**Comment by 9600bauds** [comment]
I feel like that flower should be in Theater backroom

**Comment by ghost** [comment]
I think it should be about fine. There's a few Theatre loot items scattered about. Magician outfit nearby, waterflower there, and the cape walled in

**Comment by clusterfack** [comment]
@DUNY- **GOD FUCKING DAMNIT DUNY IT'S NOT MAINT**

**Comment by Zth--** [comment]
Tl;dr but the extra morgue literally 10 tiles away from the other one has no fucking sense

**Comment by ihadtoregisterforthis** [comment]
Wow, an important map change merged under a day since published.

**Comment by gbasood** [comment]
YOU PUT FUCKING MIMIC CRATES IN

**Comment by Ephx1** [comment]
Oh Dylan, you card!

**Comment by Shadowmech88** [comment]
From the codebase that brought you maintenance bears, it's the new hit single, maintenance mimics!

**Comment by ghost** [comment]
Thank you Dylan!

**Comment by CptWad** [comment]
wow all I added was a named cockroach and here this guy goes and puts mimics in jesus christ

**Comment by Duny-** [comment]
Casual tears
I feed of em
Thx Dylan
